go/roothash/api/api.go

```diff
@@ -571,6 +571,10 @@ type ConsensusParameters struct {
 
 	// MaxEvidenceAge is the maximum age of submitted evidence in the number of rounds.
 	MaxEvidenceAge uint64 `json:"max_evidence_age"`
+
+	// MaxPastRootsStored is the maximum number of past runtime state and I/O
+	// roots that are stored in the consensus state.
+	MaxPastRootsStored uint32 `json:"max_past_roots_stored,omitempty"`
```

Comment:
Should this be `uint64`, similar as age and other stuff that depend on round which is `uint64`?

Reply:
I think `uint64` would bloat the storage way too much -- this parameter sets the max past roots stored per runtime, not globally, `uint32` should be more than enough.

Reply:
I don't think it would bloat storage as in deterministic CBOR encoding the value would be stored in the most compact way. It makes sense for this to be of the same type as the rounds.

Reply:
I didn't mean the `uint64` parameter, but the actual storage used for past rounds -- do we really need to store more than 4294967296 rounds of state and I/O roots per runtime??

Reply:
No, definitely not, I think the argument was just for it to have the same type as the round to avoid casts.

Reply:
Ah, ok, I guess I can change it to `uint64` then.